### DISCUSSION TITLE
Mapping LAMMPS type to MACE type ERROR

### DISCUSSION THREAD

**Discussion by sebastianecheverrir** [ACEsuit/mace#287] [Q&A]
I am trying to set up a simple calculation using MACE-LAMMPS and the foundation model `ani500k_large_CC.model`, after converting it to LAMMPS format.

There is an error appearing that could be related to the mapping of the atom types between LAMMPS and MACE. There seems to be something inconsistent between the atomic numbers of MACE and LAMMPS. It looks like MACE is reading the atomic number of P as 7, while LAMMPS takes it as 15. Since P is mentioned in the [paper](https://arxiv.org/abs/2312.15211), I am assuming that it is available in the potential.

Here is the full output:

```
LAMMPS (28 Mar 2023 - Development)
OMP_NUM_THREADS environment is not set. Defaulting to 1 thread. (src/comm.cpp:98)
  using 1 OpenMP thread(s) per MPI task
Reading data file ...
  orthogonal box = (0 0 0) to (48.33413 49.330816 60)
  1 by 1 by 1 MPI processor grid
  reading atoms ...
  44 atoms
  read_data CPU = 0.003 seconds
CUDA unavailable, setting device type to torch::kCPU.
Loading MACE model from "ani500k_large_CC.model-lammps.pt" ... finished.
  - The torch_float_dtype is: Double
  - The r_max is: 5.
  - The model has: 2 layers.
  - The MACE model atomic numbers are: 1 6 7 8.
  - The pair_coeff atomic numbers are: 15 8 6 1.
  - Mapping LAMMPS type 1 (P) to MACE type ERROR: Problem converting lammps_type to mace_type. (src/ML-MACE/pair_mace.cpp:403)
Last command: pair_coeff * * ani500k_large_CC.model-lammps.pt P O C H
```

Any suggestions on how to fix this error? 

Here are the files needed to reproduce the behaviour.
[4gitMace.zip](https://github.com/ACEsuit/mace/files/13874257/4gitMace.zip)

**Comment by davkovacs** [comment]
The paper you are referring to is about mace-off, the models are here: https://github.com/ACEsuit/mace-off

the model you are trying out does not have P, it is from another paper.

**Reply by sebastianecheverrir** [reply]
Thanks for the correct link to the mace-off files. As a reference for the future readers this is the [file](https://github.com/ACEsuit/mace/blob/main/mace/calculators/foundations_models/ani500k_large_CC.model) that I was mistakenly using
